.github/workflows/pipeline-workflow.yml

```diff
@@ -0,0 +1,25 @@
+name: Pipeline Workflow
+
+env:
+  REGION: us-east-1
+
+on: push
```

Comment:
Does this run on push to all branches or just specific branches?

Reply:
All branches by default, since its unknown ahead of time what repo-specific feature/main/release branches will exist